### ISSUE TITLE
Swiping and cycle questions

### ISSUE THREAD

**Issue by eypibee** [smarteist/Android-Image-Slider#63]
1. Using the default implementation of the slider, whenever I swipe it to reveal the next image, the timer is not getting reset which sometimes scroll to the next image in a speedy manner. Is this a normal behavior?

2. For instance, I have 3 images attached to the slider, whenever it reaches the last image, it passes through the second image before reaching the first. Can I hide that transition?

Thanks in advance.

**Comment by smarteist** [comment]
> 1. Using the default implementation of the slider, whenever I swipe it to reveal the next image, the timer is not getting reset which sometimes scroll to the next image in a speedy manner. Is this a normal behavior?
> 2. For instance, I have 3 images attached to the slider, whenever it reaches the last image, it passes through the second image before reaching the first. Can I hide that transition?
> 
> Thanks in advance.

Yes it's normal behavior, If you see them unloaded while passing through the middle images, you can set the 
> sliderView.setOffscreenPageLimit
to `itemsCount-1` You can get the itemsCount from the adapter , this maybe causes a memory problem in a large number of items but in your case it's ok.

**Comment by erwandy92** [comment]
To the question No. 1, I also have the same opinion on that. 
But, I still don't really get it on how to reset the time while scrolling to another page.
If it's solved by setOffScreenPageLimit, I tried it in Activity, it seems not working as expected.

Probably to set it in the Adapter?
